invenio_administration/assets/semantic-ui/js/invenio_administration/src/actions/Actions.js

```diff
@@ -55,26 +62,50 @@ export class Actions extends Component {
             />
           )}
           {displayEdit && (
-            <Button
-              as="a"
-              disabled={disableEdit}
-              href={AdminUIRoutes.editView(listUIEndpoint, resource, idKeyPath)}
-              icon
-              labelPosition="left"
-            >
-              <Icon name="pencil" />
-              Edit
-            </Button>
+            <Popup
+              content={
+                disabledEditMessage
+                  ? disabledEditMessage
+                  : i18next.t("Resource is not editable.")
+              }
+              disabled={!disableEdit}
+              trigger={
+                <span className="mr-5">
+                  <Button
+                    as="a"
+                    disabled={disableEdit}
+                    href={AdminUIRoutes.editView(listUIEndpoint, resource, idKeyPath)}
+                    icon
+                    labelPosition="left"
+                  >
+                    <Icon name="pencil" />
+                    {i18next.t("Edit")}
+                  </Button>
+                </span>
+              }
+            />
           )}
           {displayDelete && (
-            <DeleteModalTrigger
-              title={title}
-              resourceName={resourceName}
-              apiEndpoint={apiEndpoint}
-              resource={resource}
-              successCallback={successCallback}
-              idKeyPath={idKeyPath}
-              disabled={disableDelete}
+            <Popup
+              content={
+                disabledDeleteMessage
+                  ? disabledDeleteMessage
+                  : i18next.t("Resource is not deletable.")
+              }
+              disabled={!disableDelete}
+              trigger={
+                <span>
+                  <DeleteModalTrigger
+                    title={title}
+                    resourceName={resourceName}
+                    apiEndpoint={apiEndpoint}
+                    resource={resource}
+                    successCallback={successCallback}
+                    idKeyPath={idKeyPath}
+                    disabled={disableDelete}
```

Comment:
```suggestion
                    disabled={disableDelete}
                    disabledDeleteMessage={disabledDeleteMessage}
```
For accessibility:
Add `disabledDeleteMessage` to the DeleteModalTrigger and add it to the `<Element />` as `title={disabledDeleteMessage ? disabledDeleteMessage : ""}` in DeleteModalTrigger (as for the edit button).